test/test_other.py

```diff
@@ -13533,7 +13540,7 @@ def test_stdint_limits(self):
 
   def test_legacy_runtime(self):
     self.set_setting('EXPORTED_FUNCTIONS', ['_malloc', '_main'])
-    self.set_setting('DEFAULT_LIBRARY_FUNCS_TO_INCLUDE', ['$intArrayFromString', '$ALLOC_NORMAL'])
+    self.set_setting('DEFAULT_LIBRARY_FUNCS_TO_INCLUDE', ['$intArrayFromString'])
```

Comment:
This was testing that we can include either a function or a constant, I think - why remove it?

Reply:
I was thinking because its listed below in `EXPORTED_RUNTIME_METHODS` its no longer needed here (i.e. its duplicated here).     But I can revert this line.